src/formats/geotiff/GeoTiffReader.js

```diff
@@ -207,6 +218,25 @@ define([
             }
         };
 
+        /**
+         * Creates an RGB canvas from the GeoTiff image data.
+         *
+         * @return {Canvas}
+         *
+         */
+        GeoTiffReader.prototype.getImage = function () {
+            return this.createImage();
+        };
+
+        /**
+         * Creates a typed array based on the contents of the GeoTiff.
+         *
+         * @return {TypedArray}
+         */
+        GeoTiffReader.prototype.getImageData = function () {
+            return this.createTypedElevationArray();
```

Comment:
@zglueck I'm curious as to why we refer to Elevation in this class, do you know? It makes it sound like we're making an assumption about the data content. Presumably that routine predates your changes, so maybe that's a future PR.....

Reply:
GeoTiffReader's initial interface was constructed to very literally satisfy the use cases we had in mind: load an image as a Canvas, load an image as elevations.

That said, the naming doesn't isn't reflective of anything about GeoTiffReader or Tiff files. GeoTiffReader is ripe for a refactoring.